modular_skyrat/modules/goofsec/code/department_guards.dm

```diff
@@ -1,187 +1,159 @@
+	worn_icon_digi = 'modular_skyrat/master_files/icons/mob/clothing/under/engineering_digi.dmi'
 
 /obj/item/clothing/under/rank/security/officer/blueshirt/skyrat/customs_agent
 	name = "customs agent uniform"
+	desc = "A cargo-brown short-sleeve shirt, and cargo shorts in an authoritative charcoal color. Only for the FTU's finest strong-hands."
 	icon_state = "customs_uniform"
 	worn_icon_state = "customs_uniform"
-
+	icon = 'modular_skyrat/master_files/icons/obj/clothing/under/cargo.dmi'
+	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/under/cargo.dmi'
+	worn_icon_digi = 'modular_skyrat/master_files/icons/mob/clothing/under/cargo_digi.dmi'
+
+/obj/item/clothing/under/rank/security/officer/blueshirt/skyrat/bouncer
+	name = "bouncer uniform"
+	desc = "Short-sleeves and jeans, for that aura of cool that makes the drunk people listen."
+	icon_state = "bouncer"
+	worn_icon_state = "bouncer"
+	icon = 'modular_skyrat/master_files/icons/obj/clothing/under/civilian.dmi'
+	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/under/civilian.dmi'
+	worn_icon_digi = 'modular_skyrat/master_files/icons/mob/clothing/under/civilian_digi.dmi'
+
+/*
+	SUITS
+*/
 /obj/item/clothing/suit/armor/vest/blueshirt/skyrat
-	icon_state = "barney_armor"
-	worn_icon_state = "barney_armor"
-	icon = 'modular_skyrat/master_files/icons/obj/clothing/suits.dmi'
-	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/suit.dmi'
+	//Effectively the same as TG's blueshirt, including icon. The /skyrat path makes it easier for sorting.
+	icon = 'modular_skyrat/master_files/icons/obj/clothing/suits.dmi' //ORION TODO: deptguard vests need to be moved to a plain vest/skyrat subtype when suits.dmi gets split on our side
+	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/suit.dmi' //(Above) sci guard will get just the original blueshirt vest when that happens
 	unique_reskin = null
 	supports_variations_flags = CLOTHING_DIGITIGRADE_VARIATION_NO_NEW_ICON
 
-/obj/item/clothing/suit/armor/vest/blueshirt/skyrat/guard
+/obj/item/clothing/suit/armor/vest/blueshirt/skyrat/guard //Badge-less version of the blueshirt vest
 	icon_state = "guard_armor"
 	worn_icon_state = "guard_armor"
 
 /obj/item/clothing/suit/armor/vest/blueshirt/skyrat/orderly
 	name = "armored orderly coat"
-	desc = "An armored coat, to keep you safe from unruly patients."
+	desc = "An armored coat, in a deep paramedic blue. It'll keep you padded while dealing with troublesome patients."
 	icon_state = "medical_coat"
 	worn_icon_state = "medical_coat"
+	supports_variations_flags = CLOTHING_DIGITIGRADE_VARIATION
 
 /obj/item/clothing/suit/armor/vest/blueshirt/skyrat/engineering_guard
 	name = "armored engineering guard coat"
-	desc = "An armored coat, to keep you safe from unruly patients."
+	desc = "An armored coat, who's hazard strips are worn to the point of uselessness. It'll keep you protected while clearing hazard zones at least."
```

Comment:
@tf-4 would know better than me on this, but I think this would be the correct way to write it?
```suggestion
	desc = "An armored coat, whose hazard strips are worn to the point of uselessness. It'll keep you protected while clearing hazard zones at least."
```

Reply:
Valid point, wrote this at like 4am. This PR has taken way too much of my time lmao. Committing tf-4's suggestion.